src/sidebar/components/button.js

```diff
@@ -15,25 +15,41 @@ import SvgIcon from './svg-icon';
  * - `useInputStyle`: for placing an icon-only button next to an input field
  * - `usePrimaryStyle`: for applying "primary action" styling
  * - `className`: arbitrary additional class name(s) to apply
+ *
+ * Buttons have an option `actionType` value that maps the `isActive`
+ * state to the appropriate corresponding aria attribute according to the
+ * `actionType` rules defined in the propTypes def below.
  */
 export default function Button({
   buttonText = '',
   className = '',
   icon = '',
   isActive = false,
-  onClick = () => null,
-  style = {},
   title,
+  actionType,
   useCompactStyle = false,
   useInputStyle = false,
   usePrimaryStyle = false,
+  ...props
 }) {
   // If `buttonText` is provided, the `title` prop is optional and the `button`'s
   // `title` attribute will be set from the `buttonText`
   title = title || buttonText;
 
   const baseClassName = buttonText ? 'button--labeled' : 'button--icon-only';
 
+  // The `actionType` determines which aria attribute is used to
+  // express the `isActive` prop.
+  //
+  // "toggle" for aria-pressed
+  // "group" for aria-expanded
+  const ariaProp = {};
```

Comment:
For both of these attributes (`aria-pressed`, `aria-expanded`), they need to be present on the element even when logically `false`. In this current implementation, they will only be present if `true`. Perhaps use a `string` value here instead of `boolean`.